spring-web/src/main/java/org/springframework/http/client/reactive/JdkClientHttpRequest.java

```diff
@@ -59,7 +59,7 @@ class JdkClientHttpRequest extends AbstractClientHttpRequest {
 	private final HttpRequest.Builder builder;
 
 
-	public JdkClientHttpRequest(HttpMethod httpMethod, URI uri, DataBufferFactory bufferFactory,
+	JdkClientHttpRequest(HttpMethod httpMethod, URI uri, DataBufferFactory bufferFactory,
```

Comment:
Although I fully understand the desire to remove `public` here, historically the code base often makes constructors `public` even if the enclosing type is not `public`.

Look at the other `JdkClientHttpRequest` implementation as well as neighboring classes for examples.

In light of that, I removed this change when merging the PR.